src/util.c

```diff
@@ -149,6 +150,26 @@ void enable_seccomp(void)
 #endif
 }
 
+/** Use prlimit to prevent a process from forking, thus making exploitation harder */
+void forbid_fork(void)
+{
+#ifdef TARGET_OS_LINUX
+#ifdef HAVE_PRLIMIT
+  static const struct rlimit limit = {
```

Comment:
Indeed, do not even know why I did it. I removed it in the new version of the patch.